ansible/workers.yml

```diff
@@ -258,6 +328,12 @@
       - python3-cpio
       - python3-pycurl
       - python3-kickstart
+      state: installed
+    when: ansible_distribution == 'Fedora' or (ansible_distribution == 'RedHat' and ansible_distribution_version.split(".")[0] == "8")
+
+  - name: Ensure OpenSCAP Anaconda dependencies are installed (Fedora only)
```

Comment:
I would change name of task to explicitly mention zanata.
It is confusing as it is now, because there is two tasks with same name, one Fedora and RHEL8 only, and the other Fedora only.

Reply:
@matejak `yum search zanata` doesn't find anything on our RHEL 8 worker